share_extension/ios/error_message.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Text, TouchableOpacity, View} from 'react-native';
+
+import {Preferences} from 'mattermost-redux/constants';
+
+import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
+
+export default function errorMessage(props) {
+    const {close, formatMessage} = props;
+    const theme = Preferences.THEMES.default;
+    const styles = getStyleSheet(theme);
+
+    return (
+        <View style={styles.errorWrapper}>
+            <View style={styles.errorContainer}>
+                <View style={styles.errorContent}>
+                    <View style={styles.errorMessage}>
+                        <Text style={styles.errorMessageText}>
+                            {formatMessage({
+                                id: 'mobile.share_extension.error_message',
+                                defaultMessage: 'An error has occurred while using the share extension.',
+                            })}
+                        </Text>
+                    </View>
+                    <TouchableOpacity
+                        style={styles.errorButton}
+                        onPress={() => close()}
```

Comment:
change this to `onPress={close}`

Reply:
`close` is expecting a data arg which is why there is an arrow in the properties. Without the arrow in the properties an event object would get sent and that messes up the `ShareExtension.close(data, Config.AppGroupId);` since it thinks it has a valid data object. Would you prefer another function be created that doesn't take a data arg? I'm good either way, just used an arrow since these components aren't going to be re-rendered due to state changes.